nose_reportportal/plugin.py

```diff
@@ -255,8 +271,14 @@ def addError(self, test,  err):
         :param err: sys.exc_info() tuple
         :type err: 3-tuple
         """
-        test.status = "error"
-        self._addError(test, err)
+
+        if self._filterErrorForSkip(err):
```

Comment:
Could you please update the method's docstring with the description of the new logic as well?